pkg/engine/virtualmachines.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/Azure/go-autorest/autorest/to"
 )
 
-func CreateVirtualMachine(cs *api.ContainerService) VirtualMachineARM {
+func CreateMasterVM(cs *api.ContainerService) VirtualMachineARM {
```

Comment:
Renamed this func because it is (currently) only used to create the master vm. As long as that's the case having this func have a name that reflects that makes code maintenance easier.